src/formatDate.js

```diff
@@ -51,6 +51,66 @@
 
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+  let newYear = null;
+  let newMonth = null;
+  let newDay = null;
+
+  let fromYear = null;
+  let fromMonth = null;
+  let fromDay = null;
+  const result = [];
+
+  const fromDateArr = date.split(fromFormat[3]);
+
+  for (let i = 0; i < fromFormat.length; i++) {
+    const temp = (fromFormat[i][0]);
```

Comment:
Тебе не надо проверять именно первый символ, ты можешь проверять fromFormat[i], просто у тебя будет два case на YY и YYYY
```suggestion
    const datePart = fromFormat[i];
```

Reply:
Так просто добавится еще один блок кода, который, по сути ничего не меняет. В чем принципиальная разница?